app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormBranch.tsx

```diff
@@ -174,6 +174,14 @@ export const FormBranch = ({
               type="switch"
               label={featureEnabled ? "On" : "Off"}
             />
+
+            {reviewErrors !== undefined && reviewErrors.feature_enabled && (
```

Comment:
Wait can't you write this as `reviewErrors?.feature_enabled`?

Reply:
Yeah I think you should be able to:

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
right right, just wrote code while talking to my brain logically, good point @jaredlockhart  thank you :)